### ISSUE TITLE
[REQ] add number of items in folder to folder name on sync

### ISSUE THREAD

**Issue by gingerbeardman** [TeamShinkansen/Hakchi2-CE#85]
This would be useful on SNESC to see how many items are in a folder before you open it. I have folders for each RetroArch core and sometimes (depending on what I've loaded on) some of the folders are empty.

---

Folder name in Hakchi2CE: 

**Sports**

---

Folder name on SNESC after sync: 

**Sports (12)**

**Comment by ghost** [comment]
i guess your talking about the main menue.
you can actually see this on the exporting screen (at least for usb-host)

i activated this as default:
"structure" -> "custom - use folder manager"

i guess this is irritating labled, its translated better than the original english ;-)
"structure" -> "custom - folder manager start on every sync"
would be a much better understandable string ... ;-)

**Comment by gingerbeardman** [comment]
Yes, I want to see it whilst I'm using my SNESC - I should have made that more clear!
